scrunch/datasets.py

```diff
@@ -1703,9 +1703,14 @@ def create_variable(self, var_type, name, alias=None, description='',
         if categories:
             payload['categories'] = categories
         if subvariables:
-            payload['subreferences'] = [
-                {'name': item['name']} for item in subvariables
-            ]
+            payload['subreferences'] = []
+            for item in subvariables:
+                subrefs = {'name': item['name']}
+                if item.get('alias'):
+                    subrefs['alias'] = item['alias']
+                if item.get('id'):
```

Comment:
Let's remove that bit after all. Sorry!

Reply:
I was just making a point about the clarity of the code, i.e. prevent other users from expecting the `id` is actually used by Crunch when included in the payload because Crunch will ignore it regardless.

Reply:
I will proceed to remove it then, it doesn't make sense to have it there if Crunch will ignore it.